AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -109,6 +128,9 @@ private extension SearchMapView {
                 .onMarkerChange(action: { pointer in
                     viewModel.markerSelectionChanged(using: pointer)
                 })
+                .onStartingLocationChange { geoSquare in
+                    viewModel.forceUpdateOnLocationChange(geoSquare: geoSquare)
```

Comment:
this is not a view-viewmodel language. View doesn't tell viewmodel what to do, only what happened. This function is ok, but the name is incorrect, please rname